views/movies/movies.hbs

```diff
@@ -0,0 +1,6 @@
+<h2>List of Movies</h2>
+
+{{#each movieArray}}
+<h3>{{this.title}}</h3>
+<a href="/movies/{{this._id}}">See More Info</a>
+{{/each}} 
```

Comment:
Here is your route for movie list page:
```
router.get("/movies", (req, res) => {

    Movie.find()
    .then(result => {
        res.render("movies/movies", {result})
    })
    .catch((error) => {
        console.log("Error connecting to DB", error)
    });
});
```

From js to view, you are sending an object with property name `result`. However in your movies.hbs you are tryign to iterate through movieArray - this variable does not exist 🤷‍♀️ 
```
{{#each movieArray}}
```